packages/hierarchy-builder/src/hierarchy-builder/Localization.ts

```diff
@@ -0,0 +1,39 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+/** A translator implementation returns the same input. */
+const NOOP_Translator = (input: string) => input;
```

Comment:
I think this variable should be all in upper case.

Reply:
Renamed